apps/blaze-dashboard/src/load-config.js

```diff
@@ -29,7 +35,7 @@ productionConfig.features.blaze_setup_mode = !! needSetup;
 productionConfig.features[ 'use-translation-chunks' ] = false;
 
 // Sets the advertising path prefix for this app
-productionConfig.advertising_dashboard_path_prefix = isWooStore ? '/wc-blaze' : '/advertising';
+productionConfig.advertising_dashboard_path_prefix = isBlazePlugin ? '/wc-blaze' : '/advertising';
```

Comment:
One thing I noticed when forcing/hardcoding `isBlazePlugin` to `true`  is that the "Ready to promote" and "Campaigns" navigation tabs are broken and go to a 404 page because it's trying to use `wc-blaze` path. 
But I guess this won't happen in reality because the Blaze plugin would exist when this value is set to true (outside the manual testing steps).

Reply:
Yes, that is expected in the way we are testing this. I forgot to mention that, sorry.

The problem happens because we are forcing the Blaze Ads plugin dashboard, but loading it in Jetpack. Both of them uses different paths for the routes (advertising vs wc-blaze). We run some code in each of the plugins to fix those links, but because we loading one dashboard the other site, the fixing script won't work properly.

This [README.md](https://github.com/Automattic/wp-calypso/tree/trunk/apps/blaze-dashboard#routing) explains a bit why that happens. The main problem is that the app uses hashbang navigations, but some components of Calypso use root paths in links (Navlinks and Links). 

To fully test this, you will need a site running Balze Ads, and then follow the testing steps (additionally, you will need to force `isWooStore` to false to see the WPCOM theme, but links will work fine).